registry.json

```diff
@@ -15,6 +15,22 @@
     "operatorName": "",
     "github": "snic-csp"
   },
+  {
+    "ticker": "SIGMA",
+    "poolId": "8f1a90ce93850a4aa28273e31e52e71bb304a86f9cbb54b138808cf0",
+    "social": {
+      "website": "https://sigmastake.com",
+      "discord": "https://discord.gg/qSXXJAuu",
+      "reddit": "",
+      "instagram": "",
+      "facebook": "",
+      "youtube": "",
+      "twitter": "SigmaStake"
+    },
+    "telegram": "sigmastake",
```

Comment:
Please change this to be the TG handle of a user and not a group or channel

Reply:
Hi, I changed the Telegram handle now, thank you!